legend-engine-language-pure-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/PureGrammarParser.java

```diff
@@ -129,14 +136,68 @@ private PureModelContextData parse(String code, DEPRECATED_PureGrammarParserLibr
         return builder.withElement(sectionIndex).build();
     }
 
+    private PureModelContextData parse(Map<String, String> code, DEPRECATED_PureGrammarParserLibrary parserLibrary, String sourceId, int lineOffset, int columnOffset, boolean returnSourceInfo)
```

Comment:
let's just rename this method to `parseElements`, and remove the other method, also, in this particular case, does it even make sense to allow specifying `sourceId` and `lineOffset`, `columnOffset`?